jbpm-designer-api/src/main/resources/org/jbpm/designer/resources/i18n/DesignerConstants_es.properties

```diff
@@ -1,5 +1,5 @@
 # translation auto-copied from project jbpm-designer, version 6.2.0, document org.jbpm/jbpm-designer-api/org/jbpm/designer/resources/i18n/DesignerConstants, author angela.garcia
-ORYX.I18N.Language=es
+ORYX.I18N.Language=en_us
```

Comment:
Looking at https://github.com/droolsjbpm/jbpm-designer/commit/8336b837a0a72194b2f3a02c86664d1621606c94, it seems we should _not_ revert this back?